docs/core/linux-prerequisites.md

```diff
@@ -110,50 +110,13 @@ The installer bash script is used in automation scenarios and non-admin installa
 > [!IMPORTANT]
 > Before running the script, install the required [dependencies](https://github.com/dotnet/core/blob/master/Documentation/prereqs.md).
 
-## Install .NET Core for Red Hat Enterprise Linux (RHEL) 7
+## Install .NET Core for supported Red Hat Enterprise Linux (RHEL) versions
 
-To install .NET Core on RHEL 7:
-
-1. Enable the Red Hat .NET channel, available under your RHEL 7 subscription.
-    * For Red Hat Enterprise 7 Server, use:
-    
-         ```bash
-         subscription-manager repos --enable=rhel-7-server-dotnet-rpms
-         ```
-    
-    * For Red Hat Enterprise 7 Workstation, use:
-    
-        ```bash
-        subscription-manager repos --enable=rhel-7-workstation-dotnet-rpms
-         ```
-    
-    * For Red Hat Enterprise 7 HPC Compute Node, use:
-    
-        ```bash
-        subscription-manager repos --enable=rhel-7-hpc-node-dotnet-rpms
-        ```
-
-2. Install the scl tool.
-
-    ```bash
-    yum install scl-utils
-    ```
-    
-3. Install .NET Core
+To install .NET Core on supported RHEL versions:
 
 # [.NET Core 2.x](#tab/netcore2x)
 
-Install .NET Core 2.0 SDK and Runtime:
-
-   ```bash
-   yum install rh-dotnet20
-   ```
-
-Enable .NET Core 2.0 SDK/Runtime for your environment:
-
-   ```bash
-   scl enable rh-dotnet20 bash
-   ```
+To ensure you have the latest installation information, use the [.NET Core 2.x SDK and Runtime Installer instructions](https://www.microsoft.com/net/learn/get-started/linuxredhat) for supported RHEL versions.
```

Comment:
Do you really need to say "To ensure you have the latest installation information,"?

Use -> Follow

Reply:
Yeah, I'm not sure why the tabs are doing that...

Reply:
No, by oversight...